cmd/sensorbee/default_main.go

```diff
@@ -20,8 +21,8 @@ func init() {
 func main() {
 	app := cli.NewApp()
 	app.Name = "sensorbee"
-	app.Usage = "SensorBee"
-	app.Version = "0.3.2" // TODO: don't hardcode the version number
+	app.Usage = "SensorBee built with build_sensorbee 0.4.1"
```

Comment:
By the way, `"SensorBee built with build_sensorbee 0.4.1"` is generated by https://github.com/sensorbee/sensorbee/pull/63/commits/410040c6d13777e8b22f1b11b573c75c9c56ff98#diff-f31a5067e810e776393da95bccd8c0a2R199